scripts/reana

```diff
@@ -104,6 +106,91 @@ down () {
     kubectl delete -Rf configuration-manifests
 }
 
+demo () {
+    # Used to run REANA demo workflows.
+    # For now, hardcoded to run only helloworld-workflow.
+    demonstration=$2
+
+    # The code to check for flags is from http://stackoverflow.com/q/2875424
+    # Not sure if this is a good way or not.
+    flag_verbose=$(echo "$*" | grep -e "--verbose" -o)
+    flag_devel=$(echo "$*" | grep -e "--devel" -o)
+
+    if [ "$demonstration" != "" ]; then
+        #Check that there exists such a demo as inputted by user.
+        if echo "$reana_demos" | grep -qE "^$demonstration$"
+            then
+            # FIXME: Check REANA cluster is accessible. Simple ping/pong, or interaction with kubectl?
+
+            # Get URL of reana-workflow-controller
+            controller_url=$(reana get reana-workflow-monitor)
+
+            # Submit a helloworld-workflow job to workflow-controller
+            workflow_id=$(curl --fail --silent \
+                            -H "Content-Type: application/json" \
+                            -X POST -d \
+                            "{\"experiment\":\"atlas\",
+                              \"toplevel\":\"github:diegodelemos/reana-demo-helloworld@2-add-helloworld-workflow\",
+                              \"workflow\": \"hello.yaml\",
+                              \"nparallel\": \"10\",
+                              \"preset_pars\": {\"name\": \"J.Doe\", \"delay\": \"2\"}
+                            }" \
+                            http://"$(reana get reana-workflow-controller)"/yadage \
+                            | grep -oP '(?<="workflow_id": ")[^"]*')
+
+            # Check for successful execution of curl
+            # https://unix.stackexchange.com/a/204790
+            # Note that checking with '$?' might fail on some corner cases
+            # AND when authentication is involved (response codes 401 and 407)
+            submit_status=$?
+
+            if [ "$submit_status" = "0" ]; then
+
+                if [ "$flag_verbose" != "" ]; then
+                    echo "[INFO] Open http://$controller_url/$workflow_id to see visual representation of your workflow and it's progress."
+                    echo "[INFO] You can 'cat' or 'tail' /reana/atlas/$workflow_id/yadage/helloworld/greetings.txt -file"
```

Comment:
Because of reanahub/reana-workflow-engine-yadage#20 only ATLAS workflows are currently working, however, it would be nice to use something line `$(experiment)` instead of the string `atlas`. Since in `reana-resources-X` we are following the pattern `/reana/<experiment>` (see [here](https://github.com/reanahub/reana-resources-k8s/blob/master/reana_resources_k8s/templates/config.yaml#L65)) it would match perfectly